README.md

```diff
@@ -176,3 +178,27 @@ and then refactor your code from `GwtEvent` to `Event`).
    import org.gwtproject.event.legacy.shared.EventHandler;
    import org.gwtproject.event.legacy.shared.GwtEvent;
    ```
+
+## Instructions
+
+To build gwt-event:
+
+* run `mvn clean install`
+
+on the parent directory.
+
+To run the j2cl tests:
+
+* switch to the 'gwt-timer-j2cl-tests' directory
+* run `mvn j2cl:clean` & `mvn j2cl:test`
```

Comment:
should remove this, it should be run as part of install

Reply:
removed